newDB.sql

```diff
@@ -1,97 +1,110 @@
-create table Branch (
-  IFS_Code varchar(255) not null primary key,
-  Branch_Name varchar(255) not null,
-  City varchar(255) not null,
-  Branch_Address varchar(255) not null,
-  State varchar(255) not null,
-  Country varchar(255) not null,
-  PIN_Code varchar(255) not null
+create table Branches (
+  id bigserial primary key,
+  ifs_code varchar(255) not null unique,
```

Comment:
unique constraints must be added using index, this is inefficient

Reply:
PostgreSQL automatically creates a unique index when a unique constraint or primary key is defined for a table.